spec.bs

```diff
@@ -110,11 +110,13 @@ spec: CSPEE; urlPrefix: https://w3c.github.io/webappsec-cspee/
 spec: webdriver; urlPrefix: https://w3c.github.io/webdriver/
   type: dfn
     text: getting a property; url: dfn-getting-properties
+    text: undefined; url: dfn-undefined
 </pre>
 
 <pre class=link-defaults>
 spec:encoding; type:dfn; text:utf-8
 spec:infra; type:dfn; text:user agent
+spec:webdriver2; type:dfn; text:remote end steps

```

Comment:
Just FYI, private aggregation spec has a PR to remove their definition and use the webdriver's instead. Once that's landed, we can remove this in our spec.